src/menu.js

```diff
@@ -0,0 +1,50 @@
+function MenuItem(name, src) {
+  this.name = name;
+  this.src = src;
+}
+
+const menu = [
+  new MenuItem('Simple Cheese', '../src/images/chicken.jpg'),
+  new MenuItem('Mixed Foods', '../src/images/food.jpeg'),
+  new MenuItem('Nigerian Jollof Rice', '../src/images/Nigerian-Jollof.jpeg'),
+  new MenuItem('Vegetable Meal', '../src/images/veg.jpeg'),
+];
+
+const createMenuItem = ({ name, src }) => {
+  const container = document.createElement('div');
+  container.classList.add('menu-item', 'mx-auto');
+
+  const img = document.createElement('img');
+  img.src = src;
+  container.append(img);
+
+  const p = document.createElement('p');
+  p.innerText = name;
+  container.append(p);
+
+  return container;
+};
+
+const createTitle = () => {
+  const title = document.createElement('h3');
+  title.classList.add('title-header', 'text-center', 'py-4');
+  title.innerText = 'Menu';
+  return title;
+};
+
+const createMenuBackground = () => {
+  const background = document.createElement('div');
+  background.classList.add('menu-background');
+  menu.forEach((item) => {
+    background.append(createMenuItem(item));
+  });
+  return background;
+};
+
+const createMenu = (container) => {
+  // const content = document.querySelector('#content');
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history.